### ISSUE TITLE
about the possible usage of EERC

### ISSUE THREAD

**Issue by williamchern** [cpignata/eimpact-icmp#30]
“The intended field of use for the extensions defined in this document
   is administrative debugging and troubleshooting and operational
   facilities.  “
For the above description, could you please give an example of the usage of the extensions, such as would it be possible to use the EERC info for Admin to select a path with certain certifications being satisfied, in this way, Admin can choose the greenest path to transmit the traffic,etc.

**Comment by mwelzl** [comment]
Exactly, "choose a path with certain certifications being satisfied" is the use case I had in mind for this. I think that giving an example is a great idea - thanks! This can go in section 6 after the use case we already have there.

**Comment by cpignata** [comment]
Agree++!
